TESTING.md

```diff
@@ -249,6 +254,50 @@ through the browser. Expected behaviour is identical.
 * Scrollbar is **not moving automatically** while new message tokens are generated. You can easily read older messages
   without interruptions and scrolling is smooth.
 
+## Code Search
+
+All `Code Search` actions are available under the same `Sourcegraph` right-click context menu, so for simplicity, we describe only the **Expected behaviours**.
+
+To open the context menu:
+
+1. Open a file in the repository that is indexed by Sourcegraph.
+2. Select the fragment of code you want to search for (for example: `System.out.println` or `println` may be the simplest candidate).
+3. Right-click on selected fragment, navigate to the `Sourcegraph` sub-menu and choose one of the actions.
+
+### Find with Sourcegraph...
+
+// todo, it's not working for me
```

Comment:
Can you please create an issue for it if there is no one?

Reply:
https://github.com/sourcegraph/jetbrains/issues/293